### PR TITLE
Hosts count for environments

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: lib/foreman_puppet/engine.rb [hunk 1]
```diff
@@ -80,6 +80,8 @@ def argument(name, type, *rest, loads: nil, **kwargs, &block)
       ::TemplateCombination.include ForemanPuppet::Extensions::TemplateCombination
       ::ProvisioningTemplate.include ForemanPuppet::Extensions::ProvisioningTemplate
 
+      ::HostCounter.prepend ForemanPuppet::Extensions::HostCounter
```
Comment (timogoebel) [inline lib/foreman_puppet/engine.rb:83]: Should we do `include` here

and have a module Extensions that is prepended in the concern?

Comment (ezr-ondrej) [reply lib/foreman_puppet/engine.rb:83]: I don't see the value in that... if the only method we are patching needs to be prepended :thinking:

**Comment by timogoebel** [comment]
Oh, and can we have a small test for this?

File: lib/foreman_puppet/engine.rb (hunk 1 above)
Comment (timogoebel) [reply lib/foreman_puppet/engine.rb:83]: It would just be consistency. But I'm fine if we leave it like this for now.

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks!

**Comment by ezr-ondrej** [comment]
CP 1.0: c00ae5b

**Pull request merged**